build/templates/session.py/lock.py.mako

```diff
@@ -44,8 +39,4 @@
 
         Actual call to driver
```

Comment:
@marcoskirsch does this comment need to change?

Reply:
I'd get rid of the whole method

Reply:
It's still being called by the method above it, so we can't do that without removing both methods.
This sounds related to your point about some of our lock code no longer being necessary. I'd rather a change like that be handled separately.

@marcoskirsch can you create an issue, since you understand the details?

Reply:
I meant the caller could call LibraryInterpreter directly.
But yeah, there's some dead code there that only applied to a specific combination of NI-DCPower runtime, nimi-python, and using independent channels.

Reply:
Removed `_lock_session`